.travis.yml

```diff
@@ -1,14 +1,34 @@
 dist: trusty
-sudo: false
 language: node_js
 node_js:
   - "6.10.0"
+
+sudo: required
+
+services:
+  - docker
+
+env:
+ - DOCKER_COMPOSE_VERSION=1.4.2
+
 cache:
   directories:
     - ui/node_modules
 
-script: cd ui && npm install && npm test
+script: 
+  - cd ui && npm install && npm test
```

Comment:
It seems currently Tarvis builds the UI component. 
I want it to build the `components/core` also, then run the tests, and finally build the docker image using `docker build -t scorelab/Bassa components/core`